### PR TITLE
[chore] ignore generated files in packaging test for check-format

### PR DESCRIPTION
**Pull request by gtm-nayan**
Ran into this while running lint for another PR
```
@sveltejs/kit:lint: > @sveltejs/kit@1.0.0-next.354 check-format
@sveltejs/kit:lint: > prettier --check . --config ../../.prettierrc --ignore-path .gitignore
@sveltejs/kit:lint: 
@sveltejs/kit:lint: Checking formatting...
@sveltejs/kit:lint: [warn] src/packaging/test/errors/duplicate-export/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/assets/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/emitTypes-false/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/exports-merge/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/exports-replace/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/files-exclude/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/javascript/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/resolve-alias/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/fixtures/typescript/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] src/packaging/test/watch/.svelte-kit/tsconfig.json
@sveltejs/kit:lint: [warn] Code style issues found in the above file(s). Forgot to run Prettier?
@sveltejs/kit:lint:  ELIFECYCLE  Command failed with exit code 1.
@sveltejs/kit:lint: ERROR: command finished with error: command (packages/kit) pnpm run lint exited (1)
command (packages/kit) pnpm run lint exited (1)
```

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: packages/kit/.gitignore [hunk 1]
```diff
@@ -8,4 +8,8 @@
 !/src/vite/build/adapt/fixtures/*/.svelte-kit
 !/test/node_modules
 /test/apps/basics/test/errors.json
-.custom-out-dir
\ No newline at end of file
+.custom-out-dir
+
+# these are already ignored by the top level .gitignore
+# repeating them here as a faux prettier ignore
+/src/packaging/test/**/.svelte-kit
```
Comment (benmccann) [inline packages/kit/.gitignore:15]: we also have `/test/**/.svelte-kit` above. I wonder if we should combine the two into something like `**/.svelte-kit`?

Comment (gtm-nayan) [reply packages/kit/.gitignore:15]: The `**/` can probably be omitted as well, the top level gitignore already does so.

File: packages/kit/.gitignore [hunk 2]
```diff
@@ -3,9 +3,9 @@
 /assets/*
 /dist
 /docs
-/test/**/.svelte-kit
+.svelte-kit
```
Comment (benmccann) [inline packages/kit/.gitignore:6]: I liked the comment you had in the last version. Maybe move this back to the bottom and restore the comment?

**Review by benmccann — APPROVED** [review]

**Comment by benmccann** [comment]
Well, I just merged this and then had the thought that I better double check that the fixtures indeed don't use `.svelte-kit`. Perhaps this directory is a problem:

[packages/kit/src/vite/build/adapt/fixtures/basic/.svelte-kit](https://github.com/sveltejs/kit/tree/master/packages/kit/src/vite/build/adapt/fixtures/basic/.svelte-kit)

**Comment by gtm-nayan** [comment]
That one's negated on line 7 so it's not ignored, luckily it doesn't output a `package.json` so no lint errors from that folder.

**Comment by benmccann** [comment]
Ah, right! Great!

**Pull request merged**